primitives/beefy/src/mmr.rs

```diff
@@ -29,6 +29,19 @@
 use codec::{Decode, Encode};
 use scale_info::TypeInfo;
 
+/// A provider for extra data that gets added to the Mmr leaf
```

Comment:
Build is still failing.

I think you need to use `sp_std::vec::Vec` that defines the right `Vec` in both `std` and `non-std` builds. You can just `use crate::Vec` here since it's already imported in `lib.rs`.

```suggestion
use scale_info::TypeInfo;
use crate::Vec;

/// A provider for extra data that gets added to the Mmr leaf
```